### PR TITLE
studio: new settings menu ui

### PR DESCRIPTION
**Pull request by achen5671**
## Summary
https://tsu.atlassian.net/browse/IR-2867
## References
closes #_insert number here_

### CODE REVIEW

**Review by aditya-mitra — CHANGES_REQUESTED** [review]
the client settings menu is in the location route which uses the old material ui currently and not tailwind. which happens to break the ui

do we have the design already for the user menu?

image

**Comment by aditya-mitra** [comment]
merged with current dev which has tailwind planted in all routes. the select menus and tabs are non-functional and the modals appear broken

image

**Comment by achen5671** [comment]
> merged with current dev which has tailwind planted in all routes. the select menus and tabs are non-functional and the modals appear broken
> 
> image

The last time I check the select menu has always been broken. I created a separate ticket to address the issue which I will take a look at. This PR just updates the UI.

**Comment by achen5671** [comment]
> > merged with current dev which has tailwind planted in all routes. the select menus and tabs are non-functional and the modals appear broken
> > image
> 
> The last time I check the select menu has always been broken. I created a separate ticket to address the issue which I will take a look at. This PR just updates the UI.

fixed ui

Screenshot 2024-07-01 at 1 03 55 PM

**Comment by achen5671** [comment]
@aditya-mitra  I updated it to use the Modal component. I also broke out the new settings to it's own component called settings2 like we talked about. Let me know if there are any other changes you will like :)

File: packages/client-core/src/user/UserUISystem.tsx [hunk 1]
```diff
@@ -82,6 +84,7 @@ const reactor = () => {
     popupMenuState.menus.merge({
       [UserMenus.Profile]: ProfileMenu,
       [UserMenus.Settings]: SettingMenu,
```
Comment (HexaField) [inline packages/client-core/src/user/UserUISystem.tsx:86]: If the old settings menu is not being used anymore, we should remove it and replace it with this new one.

**Review by HexaField — DISMISSED** [review]

File: packages/client-core/src/user/UserUISystem.tsx (hunk 1 above)
Comment (HexaField) [reply packages/client-core/src/user/UserUISystem.tsx:86]: Oh sorry, I see that this new one is to be used in tailwind routes instead, is that correct?

**Comment by achen5671** [comment]
@aditya-mitra per up on our talk last week. Im using the MUI modal here for now to be compatible with the location route. 

Our current modal isn't compatible but in the near future we should update our current modal so that it is.

pr is ready for rereview

**Review by aditya-mitra — APPROVED** [review]
awesome 🚀

**Pull request merged**